src/patternfly/_chart-globals.scss

```diff
@@ -50,3 +51,189 @@ $pf-chart-color-black-200: $pf-color-black-400;
 $pf-chart-color-black-300: $pf-color-black-500;
 $pf-chart-color-black-400: $pf-color-black-600;
 $pf-chart-color-black-500: $pf-color-black-700;
+
+// typography
+$pf-chart-global--FontSize--xs: 12; // Axis Labels
+$pf-chart-global--FontSize--sm: 14;  // Legend
+$pf-chart-global--FontSize--2xl: 24; // Large Numbers
+$pf-chart-global--FontFamily: $pf-global--FontFamily--sans-serif;
+$pf-chart-global--letter-spacing: "normal";
+
+// label
+$pf-chart-global--label--Padding: 10;
+$pf-chart-global--label--stroke: transparent;
+$pf-chart-global--label--text-anchor: "middle";
+$pf-chart-global--label--stroke--Width: 0;
+$pf-chart-global--label--Fill: $pf-global--Color--100;
+
+// Layout Props
+$pf-chart-global--layout--Padding: 50;
+$pf-chart-global--layout--Height: 300;
+$pf-chart-global--layout--Width: 450;
+
+// Stroke
+$pf-chart-global--stroke-line-cap: "round";
+$pf-chart-global--stroke-line-join: "round";
+$pf-chart-global--stroke-dash-array: "10, 5";
+
+// Individual Charts
+
+// Area Chart
+$pf-chart-area--Opacity: .4;
+$pf-chart-area--stroke--Width: 2;
+$pf-chart-area--data--Fill: $pf-global--Color--100;
```

Comment:
There are only a handful of pf core variables that we're using here as values for chart specific vars. Seems odd that we have a mix of pf4 and chart specific vars as the values. Why wouldn't we create the same kind of system we have in core, like 

```
$pf-chart-color-black-600: $pf-color-black-900; // add this to the global colors

$pf-chart-global--Fill--Color--100: $pf-chart-color-black-600;
$pf-chart-global--Fill--Color--200: $pf-chart-color-black-200;
$pf-chart-global--Fill--Color--300: $pf-chart-color-black-300;
...
$pf-chart-global--stroke--Color--100: $pf-chart-color-black-600;
...
$pf-chart-global--warning--Color--100: $pf-chart-color-orange-300;
$pf-chart-global--warning--Color--200: $pf-chart-color-gold-300;
...
$pf-chart-global--danger--Color--100: $pf-global--danger-color--100;
...
```

Then all of the values for the chart specific variables could use chart global variables. Seems like there is a common enough pattern in the use of colors for chart properties to do something like that.

Reply:
Okay I updated the file so that there are global fill colors. For colors like black-200 and black-900 I didn't add those to the color family of 5 as these should be kept to the family of 5 to match the rest of the color sets. For the global fill variables that I just added, they reference the global colors rather than the chart color families, in case those families change.